capsules/src/proximity.rs

```diff
@@ -151,19 +151,20 @@ impl<'a> ProximitySensor<'a> {
                     return CommandReturn::success();
```

Comment:
On line 136, what happens if the current app being enqueued has the most extreme thresholds?

Reply:
yeah, in both places actually, is using unentered_grants() then you need to manually check the thresholds of the app already entered

Reply:
It's a very difficult API to use correctly.

Reply:
fixed in 13f8fd6